charts/kueue/values.yaml

```diff
@@ -153,6 +153,3 @@ webhookService:
 
 # kueue-viz dashboard
 enableKueueViz: false
-
-metrics:
```

Comment:
I would suggest to keep the default values as they are, since the change could be breaking to some workflows. I would suggest to focus this PR only on the mechanics, and the configs adjust in a follow up. Then I would be open to include the PR in RC1.

Reply:
I think this is actually addressed with the update to set 

 serviceMonitor:
    tlsConfig:
      insecureSkipVerify: true